packages/neon/neon/lib/src/widgets/unified_search_results.dart

```diff
@@ -77,15 +76,15 @@ class NeonUnifiedSearchResults extends StatelessWidget {
               ListTile(
                 leading: const Icon(
                   Icons.close,
-                  size: kAvatarSize,
+                  size: 40,
```

Comment:
And this should not be 40.
I'm for 24 (the default) but 28 is probably also ok.